pkg/virt-handler/ns/ns.go

```diff
@@ -0,0 +1,224 @@
+// Copyright 2015 CNI authors
```

Comment:
where do we use this package?

Reply:
I first thought that I need it already hear. But actually only the follow up PR #1407 starts using it.